apisix/core/utils.lua

```diff
@@ -461,5 +461,4 @@ function _M.check_tls_bool(fields, conf, plugin_name)
     end
 end
 
-
```

Comment:
This change is not necessary.

Reply:
Should I return an empty string so that the file has two empty lines at the end? Maybe this is the code style of the project and I accidentally broke it?